test/fairplay.test.js

```diff
@@ -219,7 +219,7 @@ QUnit.test('error in webkitSetMediaKeys rejects promise', function(assert) {
     }
   };
 
-  window.WebKitMediaKeys = () => {};
+  window.WebKitMediaKeys = function() {};
```

Comment:
Arrow functions aren't constructable since they are no longer transpiled into ES5. That was throwing some errors in tests.